### DISCUSSION TITLE
Not able to get the web interface in browser to login

### DISCUSSION THREAD

**Discussion by aayush1215** [Security-Onion-Solutions/securityonion#2744] [Unsupported Versions]
Version - 2.3.21
Install Source - A network installation on Ubuntu 18.04 VM on GCP cloud
Install type - Eval and Standalone (Both on different VMs)
All services are running OK in so-status
No failures when running salt-call state.highstate from the node

Issue: After installing in Ubuntu 18.04 VM in GCP with 2 interfaces (1st NIC: Manager, 2nd NIC: Monitor) and all the services running properly, I am still not able to get the web interface or the login page in the browser with the hostname I provided during installation. To create additional NIC I followed the official documentation from GCP. 

Without the login page, I am not able to perform further tasks and test my system. I am able to ping the VM in GCP.

Please help on how can I solve this problem.

Thanks in advance
Aayush Gupta

**Comment by out-n-about** [comment]
Did you run so-allow and add  your IP address to the analyst permissions?

**Reply by aayush1215** [reply]
Yes, it already exists there. I ran sudo so-allow command and chose option a) of Analyst. It shows IP address already exists.

**Reply by out-n-about** [reply]
Premissions within the cloud?  I run one in AWS and I know if the AWS Security group for the instance doesn't have the permissions correct I can't get to it either.

**Reply by aayush1215** [reply]
I have created a Virtual Private Cloud (VPC) network and also configured the relevant firewall rules to allow traffic on both the NICs. Still, the issue is the same.

**Reply by weslambert** [reply]
If you are controlling via Security Group, you can probably test using quad zero for allowing analyst access.  Additonally, I'd ensure you are honoring the method you chose to use for web access during install (hostname/ip/other).

**Comment by AndriiStepovyi** [comment]
@aayush1215  
Oh my- 2.3.21 is known for that stuff together with installation demands.
Ok- question #1: when SO Guest is booted it shows web access IP in CLI- s that IP address the same as your SO Manager interface assigned?
Question #2: >sudo so-allow is performed thus I understand, but for which IP: for the one assigned for web access or for the actual IP of SO Manager interface?

If web access IP shown on boot is wrong- you can change it:
> sudo so-ip-update
#change SO Manager web access IP

In case you need to change SO Manager/CentOS interface IP you may also use:
>sudo ntmui
#CentOS network interface config

**Reply by rmarkscott** [reply]
@aayush1215 did you get this resolved? I am having the same issue.

**Reply by kuma404** [reply]
Sudo nmtui

**Comment by jertel** [comment]
What error is the browser giving you? 

Is the hostname you entered into setup, when it asked how you will connect to the web interface, setup to resolve to the cloud instance IP?

**Reply by DinhDiesel** [reply]
Is it possible to edit the ranges I had input in the so-allow?
